### ISSUE TITLE
Year-over-year chart JS error, unknown cause

### ISSUE THREAD

**Issue by rubypollev** [apache/superset#10417]
### Expected results

The chart.

### Actual results

A JS error in the console, and no chart. 

#### Screenshots

![Screen Shot 2020-07-23 at 3 46 47 PM](https://user-images.githubusercontent.com/45610410/88345899-e2abf800-ccfb-11ea-9b8f-f7d9f3cd528b.png)

#### How to reproduce the bug

1. Create line graph of numerical data longer than 2 years
2. In Advanced Analytics run a 7-day moving average and a 52-week "actual value" difference of the metric

I have a dashboard with many of these types of graphs and some of them work, sometimes. Haven't found a pattern yet. 

### Environment

(please complete the following information):

- superset version: `0.36.0`
- python version: `3.6.10`
- node.js version: `v12.16.3`
- npm version: `6.14.4`
- OS: `macOS 10.15.5 (19F101)`
- browser: `Chrome Version 84.0.4147.89 (Official Build) (64-bit)`

### Checklist

Make sure these boxes are checked before submitting your issue - thank you!

- [X] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [X] I have reproduced the issue with at least the latest released version of superset.
- [X] I have checked the issue tracker for the same issue and I haven't found one similar.

### Additional context

JS error in console:
```
2.aca4ad85030d56974bec.chunk.js:1 Uncaught TypeError: Cannot read property 'y0' of undefined
    at 2.aca4ad85030d56974bec.chunk.js:1
    at f (explore.e9de4d7e8c553ec56a7b.entry.js:2)
    at SVGPathElement.<anonymous> (2.aca4ad85030d56974bec.chunk.js:1)
    at SVGPathElement.<anonymous> (explore.e9de4d7e8c553ec56a7b.entry.js:2)
    at explore.e9de4d7e8c553ec56a7b.entry.js:2
    at he (explore.e9de4d7e8c553ec56a7b.entry.js:2)
    at Array.J.each (explore.e9de4d7e8c553ec56a7b.entry.js:2)
    at Array.J.attr (explore.e9de4d7e8c553ec56a7b.entry.js:2)
    at SVGGElement.<anonymous> (2.aca4ad85030d56974bec.chunk.js:1)
    at explore.e9de4d7e8c553ec56a7b.entry.js:2
```

**Comment by villebro** [comment]
@rubypollev are you able to reproduce this with the example datasets? Also, have you tried with `master`? I believe fixing this should be doable (it it hasn't already been fixed in `0.37.0`) if we can only reproduce the error.

**Comment by yousoph** [comment]
Closing as we've been unable to repro. Please reopen with more details if you're still seeing this issue, thanks!
